app/assets/javascripts/application.js

```diff
@@ -51,8 +51,8 @@
 //= require controllers/container_dashboard/card/card-module
 //= require controllers/container_dashboard/card/heatmaps/heatmaps-card-directive
 //= require controllers/container_topology/container_topology_controller
-//= require d3/d3
-//= require c3/c3
+//= require c3
+//= require d3
```

Comment:
> Because the intent is to include the whole library, require c3 clearly means that. require c3/c3 means include a particular file from the library, even though it's the only one there so the end result is exactly the same.

:+1: